### ISSUE TITLE
lzh compressed streams are not decoded

### ISSUE THREAD

**Issue by sknull** [noinnion/greader#135]
I face this issue since a few days across different devices. Some news cached offline look crypted when opened. I see strange characters which look like lzh compressed data.
This occurs not for all news in a feed but for the most. Regardles which settings I make regarding user agent and offline format, etc.
I also tried to reinstall the app after having removed the cache folder.
The noinnion website appears to be offline so can we consider this app now abandonware?
The last update of this is long ago...
Any help welcome as the app is now useless for me.

Edit: In the meantime it turned out that my sdcard has a corrupt fikesystem which seems to be unrepairable. I am now commissioning an older (dead slow) card. Maybe this is issue is related to this and not to blame greader for. Once I finished my tests I will update this entry here.

**Comment by sknull** [comment]
Resolution.
To not confuse others with this issue I resolve it myself now as I have set up another sdcard.
To make it short everything now works perfectly again.

Long story:
It turned out that the image cache folder is corrupted. After trying to repair this (also using gparted to replace the garbled gpt table) it seems that the card is physically defunct and will have to be replaced.
This was a relatively new (6 month old) Sandisk Ultra 64GB card - so I never expected that it would die after such a short time (they have a ten years or so warranty for those cards).

**Comment by noinnion** [comment]
please upload some screenshots. also posts feed urls.

**Comment by sknull** [comment]
Hi,

as I remember I have mentioned in the thread that the root cause was a 
corrupted file system in greader's image cache.
Soon afterwards it turned out that the card was actually completely broken 
and was replaced by sandisk in the meantime.
TLDR: Not a greader issue this time ;)

Regards,
Stephan


--- Original-Nachricht ---
Von: noinnion
Betreff: Re: [noinnion/greader] lzh compressed streams are not decoded 
(#135)
Datum: 11.04.2016, 8:47 Uhr
An: noinnion/greader
Cc: sknull



please upload some screenshots. also posts feed urls.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/noinnion/greader/issues/135#issuecomment-208186689>
